crypto/rand/rand_lcl.h

```diff
@@ -88,7 +88,7 @@ struct rand_drbg_st {
     CRYPTO_RWLOCK *lock;
     RAND_DRBG *parent;
     int nid; /* the underlying algorithm */
-    int fork_count;
+    int fork_count; /* snapshot of global state when we last reseeded */
```

Comment:
I find this comment hard to parse.  I normally wouldn't care, but since the point of this PR is to make thngs more clear by adding comments the proposal isn't good.